test/build.py

```diff
@@ -34,6 +35,7 @@
     # unit tests
     sources = glob.glob('./unit/*/*.cpp')
     sources.extend(glob.glob('./unit/*.cpp'))
+    sources.append('../src/text/scrptrun.cpp')
```

Comment:
Why is this needed? Is it s visibility thing? In that case shouldn't the header be modified so that the right functions are visible?

Reply:
First I thought, `ScriptRun` is internal class so we shouldn't expose it. But now I've changed my mind 9e82006314da0997c576fcdffbd2777b68ec38a0 (one less file to compile)